apps/passport-server/src/util/telegramHelpers.ts

```diff
@@ -49,7 +49,7 @@ export const base64EncodeTopicData = (
 ): string => {
   const topicData = Buffer.from(
     JSON.stringify({
-      topicName,
+      topicName: encodeURIComponent(topicName),
       topicId,
       validEventIds
     }),
```

Comment:
So we should be encoding the whole topicData here. Also, why do we need base64? It seems unnecessary